modules/slugs/slugs.php

```diff
@@ -59,3 +61,10 @@ function qts_uninstall() {
     $meta_keys = "'" . implode( "','", $meta_keys ) . "'";
     $wpdb->query( "DELETE from $wpdb->postmeta WHERE meta_key IN ($meta_keys)" );
 }
+
+function qts_convert_url_before_cb( $url, $lang ) {
+    if ( empty( $url ) ) {
+        return qts_get_url( $lang );
+    }
+    return $url;
+}
```

Comment:
What happened to the check url mode != QTX_URL_QUERY?
If the result is not empty, there was a re-entring URL conversion.
I don't say we need those, but I can't judge yet if this works.

Reply:
I haven't been able to find any reason for that condition (maybe some issue superseded?), and I didn't find any issue testing with `url mode == QTX_URL_QUERY`.

Reply:
I don't know what to expect from `slugs` well enough to see if the URL mode has any impact here.